bundles/org.palladiosimulator.monitorrepository.map/model/map.ecore

```diff
@@ -61,6 +61,7 @@
     <eOperations name="isMetricNumericalBaseMetric" eType="ecore:EDataType http://www.eclipse.org/emf/2002/Ecore#//EBoolean">

```

Comment:
In this file, there are still some unused imports and an invalid set visibility that you removed in the other ecore file. Would be nice to fix this as well.